src/containers/edit-profile/profile-tab/ProfileTab.tsx

```diff
@@ -1,16 +1,123 @@
-import { FC } from 'react'
-import { ProfileProvider } from '~/context/profile-context'
-import { ProfileTabProps } from '~/types'
-import ProfileTabInnerContainer from '~/containers/edit-profile/profile-tab/profile-tab-inner-container/ProfileTabInnerContainer'
-import { getProfileContextInitialValues } from '~/utils/get-profile-context-initial-values'
+import { FC, useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Box } from '@mui/material'

```

Comment:
Use default import for `Box` instead. From [docs](https://mui.com/material-ui/guides/minimizing-bundle-size/):

![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/109922101/c4d2a72d-6d1c-4a36-a609-be3a869a2a7e)

Reply:
Thanks for noticing, changed ✅